src/pages/messaging/agent-chat-api/index.mdx

```diff
@@ -1583,6 +1583,16 @@ https://api.livechatinc.com/v3.4/agent/action/delete_event_properties \
 
 ### Tag Thread
 
+Tags a given thread. The following restrictions apply:
+
+- Active and inactive chats can be tagged.
+- One or more tags can be added to the specific **thread**.
+- To tag a chat, the **tag** needs to be created first. It can be done in the [LiveChat Application](https://accounts.livechat.com/) or via the [Configuration API](/management/configuration-api).
```

Comment:
```suggestion
- To tag a chat, the **tag** needs to be created first. It can be done in the [LiveChat Application](https://accounts.livechat.com/) or via the [Configuration API](/management/configuration-api/v2.0#tags).
```